walletlibrary/src/test/java/com/microsoft/walletlibrary/did/sdk/datasource/network/credentialOperations/FetchPresentationRequestNetworkOperationTest.kt

```diff
@@ -0,0 +1,167 @@
+package com.microsoft.walletlibrary.did.sdk.datasource.network.credentialOperations
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isInstanceOf
+import com.microsoft.walletlibrary.did.sdk.credential.service.models.oidc.PresentationRequestContent
+import com.microsoft.walletlibrary.did.sdk.credential.service.validators.JwtValidator
+import com.microsoft.walletlibrary.did.sdk.crypto.protocols.jose.jws.JwsToken
+import com.microsoft.walletlibrary.did.sdk.datasource.network.apis.ApiProvider
+import com.microsoft.walletlibrary.did.sdk.util.controlflow.Result
+import com.microsoft.walletlibrary.util.defaultTestSerializer
+import io.mockk.*
+import kotlinx.coroutines.runBlocking
+import org.junit.Test
+import retrofit2.Response
+
+class FetchPresentationRequestNetworkOperationTest {
+    private val expectedNonce = "MqTyOoBtiLhoHAVmnkXLiw=="
+    private val expectedClientId =
```

Comment:
nit: can this be like "did:test:expectedClientId"? Does it need to be a real DID?